spec/system/publishers/publishers_can_publish_a_vacancy_as_a_school_spec.rb

```diff
@@ -149,118 +161,7 @@
-      click_on I18n.t("buttons.create_job")
-      expect(current_path).to eq(organisation_jobs_start_path)
-      click_on I18n.t("buttons.create_job")
-
-      fill_in_job_title_form_fields(vacancy.job_title)
-      click_on I18n.t("buttons.save_and_continue")
-
-      fill_in_job_role_form_fields(vacancy.job_roles.first)
-      click_on I18n.t("buttons.save_and_continue")
-
-      fill_in_education_phases_form_fields(vacancy)
-      click_on I18n.t("buttons.save_and_continue")
-
-      fill_in_key_stages_form_fields(vacancy.key_stages_for_phases)
-      click_on I18n.t("buttons.save_and_continue")
-
-      fill_in_subjects_form_fields(vacancy.subjects)
-      click_on I18n.t("buttons.save_and_continue")
-
-      fill_in_contract_information_form_fields
-      click_on I18n.t("buttons.save_and_finish_later")
-
-      expect(current_path).to eq(organisation_job_path(created_vacancy.id))
-
-      has_incomplete_draft_vacancy_review_heading?(created_vacancy)
-
-      click_on I18n.t("publishers.vacancies.show.heading_component.action.complete")
-
-      expect(current_path).to eq(organisation_job_build_path(created_vacancy.id, :start_date))
-      fill_in_start_date_form_fields(vacancy.starts_on)
-      click_on I18n.t("buttons.save_and_continue")
-
-      expect(current_path).to eq(organisation_job_build_path(created_vacancy.id, :pay_package))
-
-      fill_in_pay_package_form_fields(vacancy)
-      click_on I18n.t("buttons.save_and_continue")
-
-      expect(current_path).to eq(organisation_job_build_path(created_vacancy.id, :about_the_role))
-      fill_in_about_the_role_form_fields(vacancy)
-      click_on I18n.t("buttons.save_and_continue")
-
-      expect(current_path).to eq(organisation_job_build_path(created_vacancy.id, :include_additional_documents))
-      fill_in_include_additional_documents_form_fields(vacancy.include_additional_documents)
-      click_on I18n.t("buttons.save_and_continue")
-
-      expect(current_path).to eq(organisation_job_build_path(created_vacancy.id, :school_visits))
-      fill_in_school_visits_form_fields(vacancy)
-      click_on I18n.t("buttons.save_and_continue")
-
-      expect(current_path).to eq(organisation_job_build_path(created_vacancy.id, :visa_sponsorship))
-      fill_in_visa_sponsorship_form_fields(vacancy)
-      click_on I18n.t("buttons.save_and_continue")
-
-      expect(current_path).to eq(organisation_job_build_path(created_vacancy.id, :important_dates))
-      fill_in_important_dates_form_fields(vacancy)
-      click_on I18n.t("buttons.save_and_continue")
-
-      fill_in_applying_for_the_job_form_fields(vacancy)
-      click_on I18n.t("buttons.save_and_continue")
-
-      fill_in_contact_details_form_fields(vacancy)
-      click_on I18n.t("buttons.save_and_continue")
-
-      expect(current_path).to eq(organisation_job_review_path(created_vacancy.id))
-      verify_all_vacancy_details(created_vacancy)
-    end
-
     describe "#publish" do
-      let(:yesterday_date) { Time.zone.yesterday }
```

Comment:
I added this scenario into the test of the entire flow so as to be able to reduce the number of tests.